packages/docs/src/pages/recipes/nav-bar.mdx

```diff
@@ -3,12 +3,12 @@
 
 ```jsx live=true
 <Flex
+  justifyContent="space-between"
   px={2}
   color='white'
   bg='black'
   alignItems='center'>
   <Text p={2} fontWeight='bold'>Rebass</Text>
-  <Box mx='auto' />
```

Comment:
Thanks for this, but this wouldn't work if there were more than one items before or after the Box component here, which is what this example is really intended to show

Reply:
no worries, it make sense. I was thinking on a different use case here, when you have a menu bar with left and right items nothing in the middle.